components/lib/main.dart

```diff
@@ -10,95 +11,41 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
```

Comment:
It could have been much better if it is explained what this functionality does using comments.

Reply:
please add a brief comment so its easier for other to undertand.